core/templates/pages/license-page/license-page.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="oppia-license-page">
+  <div class="oppia-static-header">
+    <h1 [innerHTML]="'I18N_LICENSE_TERMS_HEADING' | translate"></h1>
+  </div>
+  <background-banner></background-banner>
+  <div>
+    <div class="oppia-static-content-below-banner">
+      <div class="oppia-page-card oppia-static-content">
+        <div class="about-tab-container">
+          <div class="oppia-license-terms oppia-about-tab-content"
+               [ngClass]="{'oppia-about-visible-content': true}">
+            <div class="oppia-static-card-content oppia-static-card-content-narrow">
+              <h2 [innerHTML]="'I18N_LICENSE_PAGE_LICENSE_HEADING' | translate"></h2>
+              <p [innerHTML]="'I18N_LICENSE_PAGE_PARAGRAPH_1' | translate"></p>
+              <p [innerHTML]="'I18N_LICENSE_PAGE_PARAGRAPH_2' | translate"></p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="oppia-footer-padding-below-banner">
+  </div>
+</div>
+
+<style>
+  .oppia-license-page .oppia-static-content ul {
```

Comment:
I don't see any `ul` tag above, where is this used?

Reply:
removed